pombola/za_hansard/migrations/0010_create_terms_for_answers.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from pombola.south_africa.models import ParliamentaryTerm
+from pombola.za_hansard.models import Answer
+
+
+def forwards_func(apps, schema_editor):
+    # For every Answer, calculate it's term
+    for answer in Answer.objects.all():
```

Comment:
why do answers have a term as well? Does an answer's term not depend on the question's term? Can an answer exist without a term?
 I think this duplication of the field can lead to weirdness.

Reply:
I added it because answers are also checked for existing answers using the number, house and year: https://github.com/OpenUpSA/pombola/pull/11/files#diff-2267219fb444e164cfb13e1c32af4f3dR450 and because the answers table also has the unique together constraint between the number, house and year: https://github.com/OpenUpSA/pombola/pull/11/files#diff-5efde581da4ce0fd043ca07de65eef86R292-R297.

I agree that it doesn't make much sense to have those columns in the answers column though. I think it might be there because we used to get answers from multiple sources, but now we only get it from PMG and PMG already tells us which question every answer is linked to.

If we don't add the term to the answers table, we'll have to remove the unique together constraint. It might also make sense then to completely remove the number, house and year columns too.

Reply:
ah right. uhm. leave it this way for now then. not sure we need to clean it all up right now, especially after I've delayed this so long already. Let's deal with it when we see problems caused by it. I don't think it's hurting anyone now, is it?